src/man/sssd.conf.5.xml

```diff
@@ -1135,11 +1135,14 @@ fallback_homedir = /home/%u
                             <quote>password</quote> field.
                         </para>
                         <para>
-                            This option can also be set per-domain.
+                            This option can also be set per-domain (which
+                            overwrites the value in [nss] section).
                         </para>
                         <para>
                             Default: <quote>*</quote> (remote domains)
                             or <quote>x</quote> (the files domain)
+                            or <quote>x</quote> (proxy domain with nss_files
+                            and sssd-shadowutils target)
```

Comment:
I think it would be good to mention that the latter two cannot be overwritten.

Reply:
They sure can be overwritten. But only in the domain section so this should be mentioned.